### PR TITLE
Contact Page

### PR DESCRIPTION
**Pull request by danutibe07**
Mobile Portfolio : Contact Page

### CODE REVIEW

**Review by nahnahdev — COMMENTED** [review]
Hi @danutibe07 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that has an empty pull request description. A Pull Request should have a short summary describing introduced changes in more detail this is necessary to ensure that you have a professional repository. You can use this [link](https://hugooodias.medium.com/the-anatomy-of-a-perfect-pull-request-567382bb6067) to read more about writing a detailed PR description :wink:

- image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -206,15 +206,15 @@ <h2 class="about-title">Skills</h2>
...
+            <input id="fullname" name="fullname" type="text" placeholder="Full name" maxlength="30">
             <div class="error"></div>
         </div>
         <div class="input-control">
             <input id="email" name="email" type="text" placeholder="Email address">
             <div class="error"></div>
         </div>
         <div class="input-control">
-          <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here"></textarea>
+          <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" maxlength="500"></textarea>
```
Comment (iambenkis) [inline index.html]: - [ ] I was about to approve this project :clap: , this is a wonderful :pray:  However there is something you need to change. You should make sure that every section of your portfolio stick to the design it means even the position of the text should stick to the Figma design :100: . According, to picture below and kindly update your design :+1:  

### Your design

Screenshot from 2023-01-23 23-42-04

After adding a padding the following way, I got a correct design

```bash
padding: 50px;
```
### Figma design

Screenshot from 2023-01-23 23-42-31

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @danutibe07 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Owusu-Desmond) [inline index.html]: - [x] While making changes as requested by the previous code reviewer, your project design does not match the Figma design. Please make sure that you adjust the style of your project to look exactly like the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=ZV2gXsCnfUjJSp7y-0)
### Your design
Screenshot from 2023-01-23 22-20-49

### Figma design
Screenshot from 2023-01-23 22-22-49

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @danutibe07  :wave: 

You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (danutibe07) [reply index.html]: @Owusu-Desmond  This is a design for desktop view , This review was for mobile

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @danutibe07 , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Green linters
✅ ... Great design

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**